package.json

```diff
@@ -198,5 +199,17 @@
     "vuefire": "^3.2.1",
     "webpack": "^5.97.1",
     "yorkie": "^2.0.0"
+  },
+  "pnpm": {
+    "onlyBuiltDependencies": [
```

Comment:
Not sure if this should be in this PR. With pnpm v10, builds have to be approved. After you do, pnpm adds "approved builds" to `package.json`.

Reply:
it's fine, it had to be added anyway